src/main/java/lotto/service/LottoService.java

```diff
@@ -0,0 +1,58 @@
+package lotto.service;
+
+
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import lotto.domain.Lotto;
+import lotto.domain.LottoResult;
+import lotto.utils.ProfitCalculator;
+import lotto.domain.Ticket;
+import lotto.domain.WinningPrice;
+import java.util.List;
+import java.util.Map;
+import lotto.utils.RandomNumber;
+import lotto.utils.WinningCalculator;
+
+public class LottoService {
+
+    private final LottoResult lottoResult;
+    private Ticket ticket;
+
+    public LottoService(LottoResult lottoResult) {
+        this.lottoResult = lottoResult;
+    }
+
+    public int calculateTicketQuantity(int purchaseAmount) {
+        ticket = Ticket.from(purchaseAmount);
+        return ticket.getQuantity();
+    }
+
+    public List<List<Integer>> generateLottoNumber(int ticketQuantity) {
+        return IntStream.range(0, ticketQuantity)
+                .mapToObj(i -> new Lotto(RandomNumber.create()).getNumbers())
+                .collect(Collectors.toList());
+    }
+
+    public void calculateWinningStatistics(List<Integer> winningNumbers, List<List<Integer>> lottos,
+            int bonusNumber) {
+        for (List<Integer> lotto : lottos) {
+            calculatePrizeForLotto(winningNumbers, bonusNumber, lotto);
+        }
+    }
```

Comment:
```suggestion
    public void calculateWinningStatistics(
            List<Integer> winningNumbers, 
            List<List<Integer>> lottos,
            int bonusNumber) {
        for (List<Integer> lotto : lottos) {
            calculatePrizeForLotto(winningNumbers, bonusNumber, lotto);
        }
    }
```
이런식으로 좀더 깔끔하게 코드를 작성할 수 있을것 같네요!

Reply:
오 그러네요... 뭐랄까 style은 그냥 
Ctrl+Alt+L 로 정리했는데, 이런 부분은 정리가 안 되는군요.
한 번씩 점검하는 습관을 들여야겠네요. 감사합니다